content/documentation/server/gnatsd-config.md

```diff
@@ -40,9 +42,15 @@ The config file format supports the following syntax:
 - Maps can be assigned with no key separator
 - Semicolons as value terminators in key/value assignments are optional
 
+In general the configuration parameters are the same as the [command line arguments](http://nats.io/documentation/server/gnatsd-usage/). Note, however, the following differences:
+
+- The listen option is host:port for connections, on the server cli its -a and -p no hostport is supported.
+- http/https is only port on the cli, on the config it is host:port (there’s no config flag for the interface for the monitoring)
+- The -cluster flag is used for defining the host:port where routes can be solicited, on the config file this is called ‘listen’ as part property of a ‘cluster’ object.
+
 ## Sample server config file
 
-The following demonstrates an examples NATS server config file:
+The following demonstrates an example NATS server config file. See also the [NATS Server README]
```

Comment:
Not sure what of the above is causing the problem, but the following sample configuration file is not appearing in a code section (as seen with hugo).